lib/include/spi/Spi.h

```diff
@@ -306,6 +316,24 @@ namespace stm32plus {
     return true;
   }
 
+  /**
+   * Send a single half-word.
+   * @param dataToSend The hald-word to send
+   */
+  inline bool Spi::send(uint16_t dataToSend)
+  {
+      // wait for ready to send
+
+      while(!readyToSend())
+        if(hasError())
+          return false;
+
+      // send the half-word
+
+      sendData16(_peripheralAddress,*dataToSend++);
```

Comment:
yep, as I said.. my mistake, fixed it on my local branch but did not commited it right. Also a 'const' is missing after the method signature.